src/main/scala/nl.knaw.dans.easy.deposit/DataFiles.scala

```diff
@@ -65,10 +66,12 @@ case class DataFiles(bag: DansBag) extends DebugEnhancedLogging {
     }
 
     def convert(items: Map[File, String]) = {
+      implicit val pathOrdering: Ordering[Path] = Ordering.fromLessThan[Path](_.compareTo(_) < 0)
```

Comment:
From what I read here, `Path` is comparable. Hence you can derive the `Ordering` from that. You don't need this workaround with `fromLessThan`.